sdk/monitor/Microsoft.OpenTelemetry.Exporter.AzureMonitor/src/AzMonListExtensions.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Runtime.CompilerServices;
+
+namespace Microsoft.OpenTelemetry.Exporter.AzureMonitor
+{
+    internal static class AzMonListExtensions
```

Comment:
Why do these need to be extension methods?
I would expect AzMonList to have methods GetValue and GetValues as part of that class.

Reply:
Very good call out, moved this as a part of `AzMonList`

Reply:
Cool. please include some unit tests specifically for the NULL and Empty conditions. 
I saw lots of attention to those conditions in your implementation.

Reply:
I moved this method from a different file to AzMonListExtensions.cs. This PR does not change any functionality to it. I will get the test covered for this functionality in subsequent PRs.